UPGRADING.md

```diff
@@ -0,0 +1,27 @@
+# Ruby support
+
+According to [these release notes][1.9.3-eol], Ruby versions prior to `2.0.x`
+has been end-of-lifed.
+
+Additionally, this codebase makes use of [(required) keyword arguments][kwargs].
+
+From `ember-cli-rails@0.4.0` and on, we will no longer support versions of Ruby
+prior to `2.1.0`.
+
+To use `ember-cli-rails` with older versions of Ruby, try the `0.3.x` series.
```

Comment:
I'm not sure I would mention this at all. This is a `pre-1.0` project. Just let it be that you need to use the new versions.